component-interaction/src/app/services/user.service.ts

```diff
@@ -1,46 +1,56 @@
-import { Injectable } from '@angular/core';
-import { UserModel } from '../models/user.model';
+import { Injectable } from "@angular/core";
+import { UserModel } from "../models/user.model";
 
 @Injectable({
-  providedIn: 'root',
+	providedIn: "root",
 })
 export class UserService {
-    public users: UserModel[] = []
-    public constructor(){
-        this.users = [
-            {
-                "email": "alex.ionescu@gmail.com",
-                "dateOfBirth": new Date("1996-03-02"),
-                "name": "Alex Ionescu"
-            },
-            {
-                "email": "andrei.constantinescu@gmail.com",
-                "dateOfBirth": new Date("2000-07-05"),
-                "name": "Andrei Constantinescu"
-            },
-            {
-                "email": "bogdan.niculescu@gmail.com",
-                "dateOfBirth": new Date("2009-11-12"),
-                "name": "Bogdan Niculescu"
-            }
-        ]
-    }
-    public getUsers(): UserModel[]{
-        return this.users;
-    }
+	public users: UserModel[] = []
```

Comment:
```suggestion
	private users: UserModel[] = [];
```
if you have a getter, make this property private